doc/geany.txt

```diff
@@ -3357,6 +3357,10 @@ Delete to line start            Ctrl-Shift-BackSpace      Deletes from the begin
                                                           current caret position.
 
 Duplicate line or selection     Ctrl-D                    Duplicates the current line or selection.
+							  If the selection spans more than one line,
+							  this duplicates the lines containing selections.
+							  If the selection spans only a single line, then only
+							  the selection is duplicated.
 
```

Comment:
Thanks, but I'm not  sure, seems to be even more confusing to me. Why talk about the selection being multiple lines? If its the selection, its the selection.

I am not sure whats confusing about the existing description, but suggest "Duplicate the selection. If no selection duplicate the current line." if further clarification is needed.